drivers/ethernet/eth_stm32_hal.c

```diff
@@ -932,10 +932,7 @@ void HAL_ETH_ErrorCallback(ETH_HandleTypeDef *heth)
 
 	switch (error_code) {
 	case HAL_ETH_ERROR_DMA:
-		LOG_ERR("%s errorcode:%x dmaerror:%x",
-			__func__,
-			error_code,
-			HAL_ETH_GetDMAError(heth));
+		/* Do Nothing to save CPU time and avoid a log error message */
```

Comment:
Please elaborate on why saving CPU time is required in this specific case (and not on MAC error for instance)

Reply:
Many tests have been done with iperf (see attached file)
Error in HAL_ETH_ErrorCallback() function occurs when error_code == HAL_ETH_ERROR_DMA

Reply:
Iperf file attached 
[Zperf_compare_HAL_V1_V2.xlsx](https://github.com/zephyrproject-rtos/zephyr/files/10380448/Zperf_compare_HAL_V1_V2.xlsx)

Reply:
why not also saving CPU in the error below ?
`HAL_ETH_ERROR_MAC`